packages/big-design/src/components/TableNext/__snapshots__/spec.tsx.snap

```diff
@@ -303,6 +303,14 @@ exports[`renders a pagination component 1`] = `
   padding: 0.75rem 1.5rem;
 }
 
+@media (min-width:0px) {
+
+}
+
+@media (min-width:720px) {
+
+}
```

Comment:
😠 `jest-styled-components` 😠

This can't be fixed, I'm just expressing my disapproval of that library being unmaintained.